ping-thing-client.mjs

```diff
@@ -52,6 +52,18 @@ if (VERBOSE_LOG) console.log(`Starting script`);
 let USER_KEYPAIR;
 const TX_RETRY_INTERVAL = 2000;
 
+setMaxListeners(100);
+
+const mSendTransaction = sendTransactionWithoutConfirmingFactory({
+  rpc,
+});
+
+const getRecentSignatureConfirmationPromise =
+  createRecentSignatureConfirmationPromiseFactory({
+    rpc,
+    rpcSubscriptions,
+  });
```

Comment:
In general, we want to create senders/confirmers once and then use them for the lifetime of the `rpc/rpcSubscriptions`.